openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/ProfilerTab.tsx

```diff
@@ -135,7 +148,6 @@ const ProfilerTab: React.FC<ProfilerTabProps> = ({
         max: (col.max as number) || 0,
         min: (col.min as number) || 0,
         mean: col.mean || 0,
-        median: col.median || 0,
```

Comment:
Do we want to remove this?

Reply:
Yes, This was the one of requirement mentioned in the issue https://github.com/open-metadata/OpenMetadata/issues/10365